sanity.config.ts

```diff
@@ -0,0 +1,74 @@
+import { defineConfig, definePlugin } from 'sanity';
+import { deskTool } from 'sanity/desk';
+import { visionTool } from '@sanity/vision';
+import { schemaTypes } from './src';
+
+const sharedConfig = definePlugin({
+  name: 'shareConfig',
+  plugins: [deskTool(), visionTool()],
```

Comment:
Ser ut som du må importe deskStructure her på et vis: https://www.sanity.io/docs/example-migrating-the-blog-template-from-studio-v2-to-v3#277fe9219a42

I prod nå:
![image](https://github.com/navikt/familie-sanity-brev/assets/21220467/3c1c2970-ea01-404a-bd38-b162fde252cf)

Ny versjon lokal:
![image](https://github.com/navikt/familie-sanity-brev/assets/21220467/7cd10609-e0a3-4217-af81-072f32b3d5ff)

Reply:
Fikset i [Fiks desk structure](https://github.com/navikt/familie-sanity-brev/pull/401/commits/f224ee4ea10038e5a4bad57b6f27bf9c71a91980)

Reply:
Nice, så bra ut nå 👏